hermes/apps/athena/modules/staked-ada/src/config.rs

```diff
@@ -0,0 +1,21 @@
+//! Configuration constants for the staked-ada module.
+
+/// Regex pattern for matching stake address routes.
+/// Matches paths like: /api/gateway/v1/cardano/assets/stake1...
+pub const STAKE_ROUTE_PATTERN: &str = r"^/api/gateway/v1/cardano/assets/(stake1[a-z0-9]{53})$";
```

Comment:
This will match things that are not a stake address since the "bech32" explicitly excludes some characters like `1`, `b`, `i` and `o`.

I'm also not sure about the fixed prefix (`stake1`). In preprod we can have `stake_test` (see [here](https://preprod.cexplorer.io/stake/stake_test1ur4rlz6wqhczd48myqp32f98a69fa6sv7fccs3m5vveqzhcs6hyl7)).

Don't we have some proper, unit tested function for this somewhere in the libs?